subsys/bluetooth/mesh/Kconfig

```diff
@@ -181,19 +181,6 @@ config BT_MESH_PROXY_FILTER_SIZE
 	default 8 if BT_MESH_NLC_PERF_BASELINE
 	range 8 32767 if BT_MESH_NLC_PERF_BASELINE
 
-config BT_MESH_USES_TFM_PSA
```

Comment:
Since you removed it then it requires to select (probably imply) PSA_WANT_GENERATE_RANDOM for crypto provider TF-M. Mesh does not select this feature in upstream.

Reply:
It does not exist in Zephyr, PSA_WANT_GENERATE_RANDOM only exists downstream.